compiler/damlc/tests/daml-test-files/PartyCompare.daml

```diff
@@ -8,14 +8,14 @@ test = scenario do
   alice <- getParty "Alice"
   bob <- getParty "Bob"
   assert $ alice < bob
-  assert $ not (alice < alice)
-  assert $ not (bob < alice)
+  assert $ alice >= alice
```

Comment:
It looks like the intent of this test is to verify that `alice < alice` does not hold. Changing it would then be the wrong thing to do. I assume that's true for the whole file.

Reply:
Ah ok. In this instance I did change the test rather than just ignore the diagnostics. I'll fix it.